ledger/participant-integration-api/src/main/scala/platform/store/appendonlydao/events/ContractsTable.scala

```diff
@@ -50,34 +50,31 @@ private[events] object ContractsTable extends PostCommitValidationData {
       def lookup(id: ContractId): Option[Option[Instant]] =
         SQL"""
   WITH archival_event AS (
-         SELECT participant_events.*
-           FROM participant_events, parameters
+         SELECT participant_events_consuming_exercise.*
```

Comment:
Is this an independent change, or somehow it is needed for the tests to pass?
(if it is independent I would suggest not doing it in here...especially so that in the current consistent approach on the db read side using the view might give us leverage)

Reply:
It is independent, I'll remove it from this PR.

> using the view might give us leverage

Is there any advantage to using the view? Using the partitions directly seems strictly better to me (more safe and more performant).